website/project/model.py

```diff
@@ -1527,6 +1556,9 @@ def save(self, *args, **kwargs):
                 save=True,
             )
 
+            if self.creator and settings.ENABLE_NOTIFICATION_SUBSCRIPTION_CREATION:
+                self.subscribe_user_to_notifications(user=self.creator)
```

Comment:
^ This refactor isn't a blocker; feel free to make this change in a follow-up  PR